### ISSUE TITLE
alternative charger

### ISSUE THREAD

**Issue by feiltom** [andreasb242/Libreductioner#1]
hello, my charger is dead and i use your board with a led driver for charging.
thank for all, all is ok my landroid come back to life ;)

i have adapt for command on jlpcb 
https://easyeda.com/feiltom/libreductioner

i want to adapt for include charging section to the board. you say on forum 1w for 12v ?
this make 83ma ? on peak also ?
with lockdown i don t have access to my mesure device.
have a. good day.

**Comment by andreasb242** [comment]
I don't know, I have to build a device again, as I have blow up mine with a Battery Pack, which goes up to 19V by pressing the Power button a short time, and long time to turn on / off.
I pressed to short, and it completely blew up...

**Comment by feiltom** [comment]
thank you,
i make the current mesure in 2 week.
do you want have the result ?
you can close the issue.
thank you for your job.

**Comment by andreasb242** [comment]
Yes, if you have the result, post it here, because somebody may find this, because of the same question ;-)

**Comment by feiltom** [comment]
Hello, i don't have the result, no amp meter, i work again in remote.
all works for me now.
My work is here : https://github.com/feiltom/LibreWorx.
I can send you a board as thank for your job?
I make current mesure as soon as possible.

**Comment by andreasb242** [comment]
[edit: I thought I'm reply to a private notification... But don't matter]
    
      Hi
      
      Thanks for the Feedback!
      From where are you? Depending on
        this, the shipping might cost more than the Board...

      I would be much more interested in
        images, you added Charging, did you already build a case /
        connectors?
      
      By the way I use mine battery
        Powered, therefore Charging is no option ;-)
      
      My Email Address / I thought this is
        public anyway... But github uses Some anonymize address...:
      andreas_b123@bluewin.ch
Andreas
